### PR TITLE
repositories: added overlay sources, homepage and atom feed to toniz4-overlay

### PR DESCRIPTION
**Pull request by toniz4**

### CODE REVIEW

**Comment by mgorny** [comment]
Reverted due to invalid XML.

**Comment by toniz4** [comment]
sorry about that, i should have used xmllint before.

File: files/overlays/repositories.xml [hunk 1]
```diff
@@ -4900,11 +4900,15 @@
   <repo quality="experimental" status="unofficial">
     <name>toniz4-overlay</name>
     <description>Useful packages for your person use</description>
+    <homepage>github.com/toniz4/toniz4-overlay.git/</homepage>
```
Comment (mgorny) [inline files/overlays/repositories.xml:4903]: You're missing protocol too.

**Comment by toniz4** [comment]
Third time's the charm. Thanks for your patience.

**Comment by mgorny** [comment]
Sorry, lost the Closing tag on squashing.

```
commit f92580ba209fd479d17902cd4eb550536959761d
Author: Cássio <heyitscassio@cock.li>
Date:   2020-12-21 17:23:48 +0100

    repositories: added sources, homepage and atom to toniz4-overlay
    
    Signed-off-by: Michał Górny <mgorny@gentoo.org>

**Pull request closed without merging**